modules/tracking/src/gtrTracker.cpp

```diff
@@ -75,7 +75,25 @@ class TrackerGOTURNModel : public TrackerModel{
 public:
     TrackerGOTURNModel(TrackerGOTURN::Params){}
     Rect2d getBoundingBox(){ return boundingBox_; }
-    void setBoudingBox(Rect2d boundingBox){ boundingBox_ = boundingBox; }
+    void setBoudingBox(Rect2d boundingBox){
+        if ( boundingBox.x < 0 )
+        {
+            boundingBox.x = 0.0;
+        }
+        if ( boundingBox.y < 0 )
+        {
+            boundingBox.y = 0.0;
+        }
+        if ( (boundingBox.width + boundingBox.x) > image_.cols )
+        {
+            boundingBox.width = image_.cols;
+        }
+        if ( (boundingBox.height + boundingBox.y) > image_.rows )
+        {
+            boundingBox.height = image_.rows;
+        }
+        boundingBox_ = boundingBox;
+        }
```

Comment:
```
boundingBox = boundingBox & Rect(Point(0, 0), image_.size());
```

Reply:
Do you mean:
```
boundingBox_ = boundingBox & Rect(Point(0, 0), image_.size());
```
?

Reply:
Yes

Reply:
@ieliz, please don't miss this comment.

Reply:
Thank you for the reminder! I thought that I have committed these changes earlier.